### ISSUE TITLE
Make information available in machine readable format

### ISSUE THREAD

**Issue by cudeso** [enisaeu/IRtools#1]
Have the info from https://github.com/enisaeu/IRtools/blob/master/measures_and_tools.md available in a machine readable format (JSON?)

**Comment by adulau** [comment]
Yep, I just asked the same question to ENISA. Especially to create a MISP galaxy out of the document ;-)

**Comment by amicaross** [comment]
working on it
